WordPress/src/main/res/layout/backup_download_details_fragment.xml

```diff
@@ -0,0 +1,109 @@
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true"
+    android:padding="@dimen/margin_medium_large">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/backup_details_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/backup_details_action_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/backup_download_details_icon_content_description"
+            app:layout_constraintStart_toStartOf="@+id/backup_details_header"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:src="@drawable/ic_cloud_white_24dp" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/backup_details_action_button"
+            style="@style/Backup.Details.PrimaryButton"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/backup_details_description"
+            android:text="@string/backup_download_details_action_button"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/backup_details_header"
+            style="@style/Backup.Details.Header"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/backup_details_action_icon"
+            tools:text="Create downloadable backup"
+            android:text="@string/backup_download_details_header"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/backup_details_description"
+            style="@style/Backup.Details.Description"
+            app:layout_constraintStart_toStartOf="@+id/backup_details_header"
+            app:layout_constraintTop_toBottomOf="@+id/backup_details_header"
+            tools:text="October 21, 2020 at 3:40 AM is the selected point to create a downloadable backup."
+            android:text="@string/backup_download_details_description"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/backup_details_secondary_header"
+            style="@style/Backup.Details.SecondaryHeader"
+            android:layout_marginTop="13dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/backup_details_action_button"
+            tools:text="Items included in this download"
+            android:text="@string/backup_download_details_choose_items_header"/>
+
+        <!-- todo: annmarie - hmmm, I wonder if I should make this a RV, but it's hardcoded? -->
```

Comment:
Idea (💡 ): I agree with you Annmarie. Probably this owed to be a `RecyclerView` instead. I have seen such UI get out of hand. If that is to become a `RecyclerView` adding/removing/updating items would end up being much more straightforward later on.

Reply:
@ParaskP7 - exactly, once I have an entry point for the Backup feature I'll adjust. This is just for the skeleton